helm/content-store/values.yaml

```diff
@@ -10,17 +10,21 @@ service:
   annotations: {}
   type: NodePort
   port: 80
-  targetPort: 80
-
 
 replicaCount: 1
 
-image:
+appImage:
   repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
   pullPolicy: IfNotPresent
   tag: latest
 
-containerPort: 80
+nginxImage:
+  repository: nginx
+  pullPolicy: IfNotPresent
```

Comment:
If you switch from `latest` to `stable` (or anything else for that matter) then you'll no longer need to specify this (i.e. it'll default to `IfNotPresent`).
```suggestion
```

Reply:
Sorry didn't think this through properly - if we're using any label that can move around we don't really want IfNotPresent because there's no way for it to know to re-pull when the label moves.

Reply:
This is very much something for another day, but as any Docker tag can arbitrarily move around the only 100% safe usage of `IfNotPresent` is when using a container's image digest (`image@sha256:`) rather than a tag - definitely not something we need to get into now though.